tools/Customization/DevHome.Customization/ViewModels/FileExplorerViewModel.cs

```diff
@@ -37,6 +42,21 @@ public FileExplorerViewModel()
             new(stringResource.GetLocalized("MainPage_Header"), typeof(MainPageViewModel).FullName!),
             new(stringResource.GetLocalized("FileExplorer_Header"), typeof(FileExplorerViewModel).FullName!)
         ];
+
+        var experimentationService = Application.Current.GetService<IExperimentationService>();
+        if (experimentationService.IsFeatureEnabled("FileExplorerSourceControlIntegration"))
+        {
+            // Currently, the UI displays a drop down which allows the user to select from a list of source control providers avaiable to Dev Home. This is
```

Comment:
I've read "Subject to change" three times.  When will the UI decided upon, should the PR wait until that happens?